pdns/dnsdistdist/docs/advanced/axfr.rst

```diff
@@ -24,6 +27,25 @@ and moving the source address check to :program:`dnsdist`'s side::
   Before 1.4.0, the QTypes were in the ``dnsdist`` namespace. Use ``dnsdist.AXFR`` and ``dnsdist.IXFR`` in these versions.
   Before 1.4.0, the RCodes were in the ``dnsdist`` namespace. Use ``dnsdist.REFUSED`` in these versions.
 
+A different way would be to configure dnsdist to pass the source IP of the client to the backend. The different options
+to do that are described in :doc:`Passing the source address to the backend <passing-source-address>`.
+
+.. warning::
+
+  Be wary of dnsdist caching the responses to AXFR and IXFR queries and sending these to the wrong clients.
+  This is mitigated by default when the source IP of the client is passed using EDNS Client Subnet, but
+  not when the proxy protocol is used, so disabling caching for these kinds of queries is advised:
+
+  .. code-block:: lua
+
+    -- this rule will not stop the processing, but disable caching for AXFR and IXFR responses
+    addAction(ORRule({QTypeRule(DNSQType.AXFR), QTypeRule(DNSQType.IXFR)}), SetSkipCacheAction())
```

Comment:
what would dnsdist be caching? not a whole zone transfer I hope? but perhaps a short REFUSED answer?

Reply:
```suggestion
    addAction(OrRule({QTypeRule(DNSQType.AXFR), QTypeRule(DNSQType.IXFR)}), SetSkipCacheAction())
```

Reply:
dnsdist only caches answers smaller or equal to 4096 bytes, so hopefully not the full AXFR response, although in theory it would be possible. Perhaps we should explicitly skip that?

Reply:
Yes, I think hardcoding that makes sense!